prov/efa/src/rxr/rxr.h

```diff
@@ -188,6 +188,7 @@ static inline void rxr_poison_mem_region(uint32_t *ptr, size_t size)
 #define RXR_SHM_HDR_DATA	BIT_ULL(11)
 
 extern struct fi_info *shm_info;
+extern int efa_cma_cap;
```

Comment:
Can we move this variable to inside rxr_env? also, shouldn't it named shm_cma_cap because EFA does not use CMA?

Reply:
Sure, will name it as shm_cma_cap.